sdk/program/src/sysvar/epoch_rewards.rs

```diff
@@ -43,9 +51,14 @@
 //! #
 //! # use solana_program::sysvar::SysvarId;
 //! # let p = EpochRewards::id();
-//! # let l = &mut 1120560;
-//! # let d = &mut vec![0, 202, 154, 59, 0, 0, 0, 0, 10, 0, 0, 0, 0, 0, 0, 0, 42, 0, 0, 0, 0, 0, 0, 0];
-//! # let a = AccountInfo::new(&p, false, false, l, d, &p, false, 0);
+//! # let l = &mut 1559040;
+//! # let mut epoch_rewards = EpochRewards::default();
+//! # epoch_rewards.distribution_starting_block_height = 42;
+//! # epoch_rewards.total_rewards = 100;
+//! # epoch_rewards.distributed_rewards = 10;
+//! # epoch_rewards.active = true;
```

Comment:
micro-nit: is there a reason to construct these differently from the normal tests? ie as
```
    let epoch_rewards = EpochRewards {
            distribution_starting_block_height: 42,
            total_rewards: 100,
            distributed_rewards: 20,
            active: true,
            ..EpochRewards::default()
        };
```

Reply:
No good reason! Technically, these lines are hidden from docs viewers, but I will update so that it's cleaner for code-readers.

Reply:
Pushed